app/controllers/org_admin/questions_controller.rb

```diff
@@ -70,6 +70,8 @@ def update
         # modifiable (versioned) question
         attrs = question_params
         attrs = transfer_associations(question) if question.id != params[:id]
+        # If the user unchecked all of the themes set the association to an empty array
+        attrs[:theme_ids] = [] unless attrs[:theme_ids].present?
```

Comment:
nice check here. Better to add it at the server level that relying on setting it up through JS .